Dockerfile

```diff
@@ -123,9 +121,15 @@ FROM base as dev
 # Swap user, so the following tasks can be run as root
 USER root
 
+# Install `psql`, useful for `manage.py dbshell`, and dependencies for installing nodejs
+RUN apt-get update --yes --quiet && apt-get install --yes --quiet --no-install-recommends \
+    postgresql-client \
+    ca-certificates \
+    gnupg \
+    && apt-get autoremove && rm -rf /var/lib/apt/lists/*
```

Comment:
suggestion (non-blocking): It doesn't matter that much for the dev image, but this last line will slow down the next `apt-get update`, and because you're not doing the same thing after the `nodejs` update, the final image will be a bit bigger.

You could both `apt-get` calls, and the commands to add the `nodesource` repo in to a single `RUN` command to keep this layer quite small, but up to you whether you think that's worthwhile.

Reply:
That makes total sense, thanks! I've moved that line to after the node installation